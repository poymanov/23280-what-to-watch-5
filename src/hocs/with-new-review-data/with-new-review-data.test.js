import React from "react";
import renderer from "react-test-renderer";
import withNewReviewData from "./with-new-review-data";


const MockComponent = () => {
  return (
    <div />
  );
};

const MockComponentWrapped = withNewReviewData(MockComponent);

it(`withNewReviewData is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped id="1">
      <React.Fragment />
    </MockComponentWrapped>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
