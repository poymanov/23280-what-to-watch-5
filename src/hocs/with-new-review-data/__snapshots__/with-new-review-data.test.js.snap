// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withNewReviewData is rendered correctly 1`] = `<div />`;
